### PR TITLE
[DataStorages] Add privilege guide in prerequisites section

### PR DESCRIPTION
**Pull request by yunmiha**
Signed-off-by: Yunmi Ha <yunmi.ha@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Add privilege guide in prerequisites section

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - N/A

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - N/A

### CODE REVIEW

File: docs/application/native/guides/data/data-storages.md [hunk 1]
```diff
@@ -17,11 +17,20 @@ The main features of the Storage API include:
...
-```
+1. To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+
+   ```
+   #include <storage.h>
+   ```
+
+2. To access files or directories in storage, you must declare following privileges:
+
+   - For the internal storage, declare `http://tizen.org/privilege/mediastorage`
```
Comment (Reni-90) [inline docs/application/native/guides/data/data-storages.md]: Suggestion:
This line is fine. However, to maintain consistency with other pages can we write as:
```
<privileges>
   <privilege>http://tizen.org/privilege/mediastorage</privilege>
</privileges>
```

For more information, see:
1. https://developer.tizen.org/development/guides/web-application/data-storage-and-management/stored-content-management
2. https://developer.tizen.org/development/guides/native-application/connectivity-and-wireless/bluetooth

Please verify and share your opinion.

@mijin85cho 
Please verify and share your valuable opinion regarding the same.

Comment (mijin85cho) [reply docs/application/native/guides/data/data-storages.md]: Good checking, @Reni-90. 
The author updated the PR. Would you please check the latest?

Comment (Reni-90) [reply docs/application/native/guides/data/data-storages.md]: @mijin85cho 
Thank you. Sure, I will.

File: docs/application/native/guides/data/data-storages.md [hunk 2]
```diff
@@ -17,11 +17,25 @@ The main features of the Storage API include:
...
+1. To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+
+   ```
+   #include <storage.h>
+   ```
+
+2. To access the files or directories of the device storage, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+   ```
+   <privileges>
+      <!--For the internal storage-->
```
Comment (mijin85cho) [inline docs/application/native/guides/data/data-storages.md]: Only one more point that I almost missed.

Please use 4 spaces for the indentation.
We need 1 more space from line 31 to line 34.

Comment (yunmiha) [reply docs/application/native/guides/data/data-storages.md]: @mijin85cho I updated about this. please review it again. Thank you.

**Comment by mijin85cho** [comment]
The update is applied to Tizen Developer Site also.

**Pull request merged**